extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -101,6 +101,33 @@
     @ConfigItem
     Optional<String> implicitNamingStrategy;
 
+    /**
+     * Defines the method for multi-tenancy (DATABASE, NONE, SCHEMA). The complete list of allowed values is available in the
+     * https://docs.jboss.org/hibernate/stable/orm/javadocs/org/hibernate/MultiTenancyStrategy.html[Hibernate ORM JavaDoc].
+     * The type DISCRIMINATOR is currently not supported.
+     *
+     * @asciidoclet
+     */
+    @ConfigItem
+    public Optional<String> multitenant;
+
+    /**
+     * Defines the name of the data source to use in case of SCHEMA approach. The default data source will be used if not set.
+     *
+     * @asciidoclet
+     */
+    @ConfigItem
+    public Optional<String> multitenantSchemaDatasource;
+
+    /**
+     * Defines if the tenant identifier of the "current sessions" should be validated every time the tenant identifier is
```

Comment:
I'm wondering if it wouldn't be better to refrain from exposing this option. AFAIR Quarkus we don't expose `org.hibernate.context.spi.CurrentSessionContext`, which makes this check useless?

I'd prefer to omit this, as less is more - especially when it's easy to add things, but never easy to remove something.

Also, removing this configuration option would resolve a couple more comments I had :)

Reply:
OK, I removed it